Gruntfile.js

```diff
@@ -173,6 +193,7 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-prompt');
   grunt.loadNpmTasks('grunt-bump');
   grunt.loadNpmTasks('grunt-npm');
+  grunt.loadTasks('node_modules/freedom-for-chrome/tasks');
```

Comment:
As of today, we have separated this out into a separate repo/npm module
https://www.npmjs.com/package/grunt-jasmine-chromeapp

Let's use that instead